src/oas3/transformers/__tests__/__snapshots__/responses.test.ts.snap

```diff
@@ -48,9 +46,13 @@ Array [
             "value": 1000,
           },
         ],
+        "explode": false,
         "name": "fake-header-name-1",
         "schema": Object {
+          "$schema": "http://json-schema.org/draft-07/schema#",
           "format": "int32",
+          "maximum": 2147483647,
+          "minimum": -2147483648,
```

Comment:
🤔 ?

Reply:
long story short - was a bug to clobbered typings. We've never "translated" schemas located under headers in responses. These values are inserted by the aforementioned converter. 
`int32` format is specific to OAS, so we got to convert the values if we want to be compliant with JSON Schema and have `exclusiveMinimum` etc. handled properly.
https://github.com/stoplightio/http-spec/blob/master/src/oas/transformers/schema/keywords/format.ts#L24

Reply:
Hmm, not sure this is practical though... with this, everybody is going to see these random maximum/minimum values in their docs, etc.